thunder/torch/__init__.py

```diff
@@ -583,7 +602,7 @@ def full_like(
 @torchsymbol(torch.ones)
 def ones(*shape: int, device: None | DeviceLike = None, dtype: None | dtypeLike = None) -> TensorLike:
     shape = utils.extract_shape_from_varargs(shape)
-    return full(shape, 1, device=device, dtype=dtype)
+    return full(shape, 1.0, device=device, dtype=dtype)
```

Comment:
I think we have to explicitly specify the `dtype` here as `maybe_get_default_dtype(dtype))`, so that `full` will not try to infer the value from `fill_value`. (similar for `zeros`)

```suggestion
    return full(shape, 1, device=device, dtype=maybe_get_default_dtype(dtype))
```